operators/pkg/tenant-controller/sandbox_test.go

```diff
@@ -0,0 +1,253 @@
+			Expect(reconciler.Get(ctx, sandboxNSname, &sbNamespace)).To(Succeed())
+			for k, v := range forge.SandboxObjectLabels(nil, tenantName) {
+				Expect(sbNamespace.GetLabels()).To(HaveKeyWithValue(k, v))
+			}
+			Expect(sbNamespace.GetOwnerReferences()).To(ContainElement(ownerRef))
+		})
+
+		It("Should fill the correct sandbox status value", func() {
+			Expect(tenant.Status.SandboxNamespace).To(BeIdenticalTo(presenceStatus))
+		})
+	}
+
+	DescribeSandboxResQuotaPresence := func() {
+		It("Resource quota should be present and have the expected labels", func() {
+			Expect(reconciler.Get(ctx, resQuotaNSname, &sbResQuota)).To(Succeed())
+			for k, v := range forge.SandboxObjectLabels(nil, tenantName) {
+				Expect(sbResQuota.GetLabels()).To(HaveKeyWithValue(k, v))
+			}
+			Expect(sbResQuota.GetOwnerReferences()).To(ContainElement(ownerRef))
+
+		})
+
+		It("Resource quota should be present and have the expected spec", func() {
+			Expect(reconciler.Get(ctx, resQuotaNSname, &sbResQuota)).To(Succeed())
+			for k, v := range forge.SandboxResourceQuotaSpec() {
+				Expect(sbResQuota.Spec.Hard).To(HaveKeyWithValue(k, WithTransform(
+					func(q resource.Quantity) string { return q.String() }, Equal(v.String()))))
+			}
+
+		})
+	}
+	DescribeSandboxRoleBindPresence := func() {
+		It("Role binding should be present and have the expected labels", func() {
+			Expect(reconciler.Get(ctx, roleBindingNSname, &sbRoleBinding)).To(Succeed())
+			for k, v := range forge.SandboxObjectLabels(nil, tenantName) {
+				Expect(sbRoleBinding.GetLabels()).To(HaveKeyWithValue(k, v))
+			}
+			Expect(sbRoleBinding.GetOwnerReferences()).To(ContainElement(ownerRef))
+
+		})
+
+		It("Role binding should be present and have the expected roleref", func() {
+			Expect(reconciler.Get(ctx, roleBindingNSname, &sbRoleBinding)).To(Succeed())
+			Expect(sbRoleBinding.RoleRef).To(BeIdenticalTo(roleRef))
+		})
+
+		It("Role binding should be present and have the expected subjects", func() {
+			Expect(reconciler.Get(ctx, roleBindingNSname, &sbRoleBinding)).To(Succeed())
+			for i, subject := range subjects {
+				Expect(sbRoleBinding.Subjects[i]).To(BeIdenticalTo(subject))
+			}
+		})
+
+	}
+
+	DescribeSandboxNsAbsence := func() {
+		It("Should set the sandbox status empty", func() {
+			Expect(tenant.Status.SandboxNamespace).To(BeIdenticalTo(emptyStatus))
+		})
+
+		It("Namespace should not be present", func() {
+			Expect(reconciler.Get(ctx, sandboxNSname, &sbNamespace)).To(FailBecauseNotFound())
+		})
+
+	}
+
+	BeforeEach(func() {
+		ctx = ctrl.LoggerInto(context.Background(), logr.Discard())
+		clientBuilder = *fake.NewClientBuilder().WithScheme(scheme.Scheme)
+		InitializeTestStructs()
+	})
+
+	JustBeforeEach(func() {
+		client := clientBuilder.Build()
+		reconciler = controllers.TenantReconciler{Client: client, Scheme: scheme.Scheme}
+		ctx, _ = clctx.TenantInto(ctx, &tenant)
+		err = reconciler.EnforceSandboxResources(ctx, &tenant)
+
```

Comment:
Be consistent in formatting. Remove useless newlines at the end of methods, and use newlines to highlight different blocks.